.github/ansible/redeploy-derper.yml

```diff
@@ -1,20 +1,20 @@
 ---
-- name: Update derper node
+- name: Update iroh-relay node
   hosts: derper
   become: yes
 
   tasks:
-  - name: Fetch derper binary
+  - name: Fetch iroh-relay binary
     get_url:
       url: https://vorc.s3.us-east-2.amazonaws.com/derper-linux-amd64-{{ derper_version }}
       mode: '0755'
       force: yes
-      dest: /usr/local/bin/derper
+      dest: /usr/local/bin/iroh-relay
```

Comment:
for the cheat code mode we still need this and te line bellow to be `derper` as I need to do the infra adjustments too.